api/hypercore/convert_to_rowstore.md

```diff
@@ -0,0 +1,90 @@
+---
+api_name: convert_to_rowstore()
+excerpt: Move a chunk from the columnstore to the rowstore
+topics: [columnstore, hypercore]
+keywords: [columnstore, hypercore, rowstore, chunks, backfilling]
+api:
+  license: community
+  type: function
+---
+
+import Since2180 from "versionContent/_partials/_since_2_18_0.mdx";
+
+# convert_to_rowstore() <Tag type="community">Community</Tag>
+
+Manually convert a specific chunk in the hypertable columnstore to the rowstore.
+
+If you need to modify or add a lot of data to a chunk in the columnstore, best practice is to stop 
+any [jobs][job] moving chunks to the columnstore, convert the chunk back to the rowstore, then modify the 
+data. After the update, [convert the chunk to the columnstore][convert_to_columnstore] and restart the jobs.
+This workflow is especially useful if you need to backfill old data.
+
+<Since2180 />
+
+## Samples
+
+To modify or add a lot of data to a chunk:
+
+<Procedure>
+
+1. **Stop the jobs that are automatically adding chunks to the columnstore**
+   
+   Retrieve the list of jobs from the [timescaledb_information.jobs][informational-views] view
+   to find the job you need to [alter_job][alter_job].
+
+   ``` sql
+   SELECT alter_job(JOB_ID, scheduled => false);
+   ```
+
+1. **Convert the chunks to update back to the rowstore**
+
+   - Convert single chunk:
+
+      ``` sql
+      SELECT convert_to_rowstore('_timescaledb_internal._hyper_2_2_chunk');
+      ```
+
+   - Convert all chunks in a hypertable named `metrics`:
+
+      ``` sql
+      SELECT convert_to_rowstore(c, true) FROM show_chunks('metrics') c;
+      ```
```

Comment:
Since `convert_to_rowstore` is a procedure, this type of query is no longer supported. 

It is possible to write a wrapper procedure to convert all chunks, like this, but maybe we should just remove this from the docs for now? Normally, users rely on a policy to convert in any case.

Reply:
Updated, thank you.